build/commands/lib/config.js

```diff
@@ -54,11 +54,20 @@ var packageConfigBraveCore = function (key) {
 
 const getNPMConfig = (key) => {
   if (!NpmConfig) {
-    const list = run(npmCommand, ['config', 'list', '--json', '--userconfig=' + path.join(rootDir, '.npmrc')])
```

Comment:
The file doesn't exit in brave-browser (was removed a few month ago), but it still used in devops repo to pass variable to the build ;(

Reply:
Changing `--userconfig` to `cwd` allow to use default userconfig (that is usually used by developers)
CI will continue to use `brave-browser/.npmrc` and it will take a priority over `~/.npmrc` as expected